source/utils.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
         outputPath: path.join(__dirname, '..', 'static'),
         withCoverage: process.argv.indexOf('--with-acceptance') === -1 && process.argv.indexOf('--no-coverage') === -1,
         withLint: process.argv.indexOf('--no-lint') === -1,
+        withFix: process.argv.indexOf('--fix') > -1,
```

Comment:
Just a thought. I know that this is `eslint` syntax. But since we have a lot of stuff going on here, maybe we can be more explicit, like `--fix-eslint` (`--fix-lint`)? Or would it be too much?